config/features.yml

```diff
@@ -1294,8 +1290,6 @@ features:
   virtual_agent_enable_nlu_pva_testing:
     actor_type: user
     description: If enabled, allows for connecting to NLU PVA
-  vye_request_allowed:
+  vye_load_scrypt_config:
```

Comment:
@adamwhitlock1  The feature flipper`vye_request_allowed` is present in the following files:

1. modules/vye/app/controllers/vye/v1/application_controller.rb
2. modules/vye/spec/requests/vye/v1/show_spec.rb

Reply:
Not sure that got out of sync, but should be fixed now. Thanks